src/js/rx/containers/Main.jsx

```diff
@@ -12,15 +12,13 @@ class Main extends React.Component {
   render() {
     let alertBox;
 
-    if (this.props.alert.visible) {
-      alertBox = (
-        <AlertBox
-            content={this.props.alert.content}
-            isVisible={this.props.alert.visible}
-            onCloseAlert={this.props.closeAlert}
-            status={this.props.alert.status}/>
-      );
-    }
+    alertBox = (
+      <AlertBox
```

Comment:
Is it ok for us to use this? There are some css classes prefixed with rx in the code for it.